x-pack/test/functional/apps/dashboard/drilldowns/index.ts

```diff
@@ -26,7 +26,9 @@ export default function ({ loadTestFile, getService }: FtrProviderContext) {
 
     loadTestFile(require.resolve('./dashboard_to_dashboard_drilldown'));
     loadTestFile(require.resolve('./dashboard_to_url_drilldown'));
-    loadTestFile(require.resolve('./explore_data_panel_action'));
+    // Disabled for now as it requires xpack.discoverEnhanced.actions.exploreDataInContextMenu.enabled
+    // setting set in kibana.yml to work. Once that is enabled by default, we can re-enable this test suite.
+    // loadTestFile(require.resolve('./explore_data_panel_action'));
```

Comment:
Can't you just add the parameter in the FTR config for now?

https://github.com/elastic/kibana/blob/5a6eda2b2210c63c0d9bb5a14510e63dd35b2b0b/x-pack/test/functional/config.js#L78-L88

Reply:
Good point, pushed a fix